dashboard/lib/services/lti.rb

```diff
@@ -42,4 +42,107 @@ def self.get_claim_from_list(id_token, keys_array)
   def self.get_claim(id_token, key)
     id_token[key] || id_token.dig(Policies::Lti::LTI_CUSTOM_CLAIMS, key)
   end
+
+  def self.initialize_lti_student_from_nrps(client_id:, issuer:, nrps_member:)
+    # TODO DAYNE don't do [0]
+    custom_claims = nrps_member[:message][0][Policies::Lti::LTI_CUSTOM_CLAIMS.to_sym]
+    user = User.new
+    user.provider = User::PROVIDER_MIGRATED
+    user.user_type = User::TYPE_STUDENT
```

Comment:
Will there ever be a case where there may be a co-teacher in the nrps_member list, and we would want to create a teacher account for them?

Reply:
Yeah there are actually a few different LTI user types that might be considered co-teachers in Code.org, so we'll definitely want to add this functionality before we go GA. We have a [Jira ticket open](https://codedotorg.atlassian.net/browse/P20-603) for this, but it's just a placeholder for now.